addOns/oast/src/main/java/org/zaproxy/addon/oast/services/boast/BoastOptionsPanelTab.java

```diff
@@ -94,8 +94,7 @@ ZapTextField getBoastUri() {
     ZapNumberSpinner getPollingFrequencySpinner() {
         if (pollingFrequencySpinner == null) {
             pollingFrequencySpinner =
-                    new ZapNumberSpinner(
-                            BoastParam.MINIMUM_POLLING_FREQUENCY, 60, Integer.MAX_VALUE);
+                    new ZapNumberSpinner(BoastParam.MINIMUM_POLLING_FREQUENCY, 60, 6 * 60 * 60);
```

Comment:
I know it makes it a bit more verbose, but it's more obvious for maintenance etc.
```suggestion
                    new ZapNumberSpinner(BoastParam.MINIMUM_POLLING_FREQUENCY, TimeUnit.HOURS.toSeconds(6));
```